config/initializers/sufia_catalog_search_builder_overrides.rb

```diff
@@ -18,4 +18,9 @@
     klass.send(:include, SearchBuilder::RestrictAdminSearchFields)
   end
 
+  unless klass.ancestors.include? SearchBuilder::SyntheticCategoryLimit
+    klass.send(:include, SearchBuilder::SyntheticCategoryLimit)
+    klass.synthetic_category_param = :temp_synthetic_collection
```

Comment:
The idea is that the module is included, but inactive waiting to be activated, everywhere. But then it can be activated with the params discussed above. 

Hydra makes it confusing when the `::SearchBuilder` is in use, and when other search builders are in use. I thought it made sense to include the module "everywhere", so when you needed it you could just activate it without having to figure out "what search builder is in use now, and was the module included in it?"  Cause it's confusing. 

In general, when I'm adding things to the special catalog search builder through this weird hack, I'm also adding the to `::SearchBuilder` -- jcoyne has a PR in to make `::SearchBuilder` be the ancestor of _everything_, if that happens, we can get rid of the weird override hack and just use `::SearchBuilder`. 

See https://github.com/projecthydra-labs/hyrax/issues/707
https://github.com/projecthydra-labs/hyrax/pull/769